src/rard/templates/research/partials/rich_text_editor.html

```diff
@@ -2,7 +2,7 @@
 {# assumes form field as a context variable named 'field' and whether to enable mentions 'enable_mentions' and optional min_height and padding_bottom in pixels #}
 
 <div class='row mb-3 mt-n3 editor-area'>
-    <div class='col h-fill' style='min-height: {% if min_height %}{{ min_height }}{% else %}400{% endif %}px; padding-bottom: {% if padding_bottom != None %}{{ padding_bottom }}{% else %}60{% endif %}px;'>
+    <div class='col h-fill {% if extra_classes %}{{extra_classes}}{% endif %}' style='min-height: {% if min_height %}{{ min_height }}{% else %}400{% endif %}px; max-height: {% if max_height %}{{ max_height }}px{% else %}auto{% endif %}; padding-bottom: {% if padding_bottom != None %}{{ padding_bottom }}{% else %}60{% endif %}px;'>
```

Comment:
Just wondering at what point it's better to put the min-height, max-height, padding stuff in css and just apply a class?

Reply:
Like just apply a classname? That would be better practice, I think. I can open an issue for it to be done at some point